src/config.rs

```diff
@@ -49,6 +51,7 @@ impl Config {
             workspace_mode: false,
             analyze_package: None,
             clear_env_rust_log: true,
+            build_on_save: true,
```

Comment:
Ohh I don't believe it, in the comment I have written false and then I write true. It was clearly too tired for programming ^^